apps/docs/pages/developer/get-request/first-get-request.mdx

```diff
@@ -29,10 +29,10 @@ touch 0rbit-Get-Request.lua
 local json = require("json")
 
 _0RBIT = "BaMK1dfayo75s3q1ow6AO64UDpD9SEFbeE8xYrY2fyQ"
-_0RBT_TOKEN = "BUhZLMwQ6yZHguLtJYA5lLUa9LQzLXMXRfaq9FVcPJc"
+_0RBT_POINTS = "BUhZLMwQ6yZHguLtJYA5lLUa9LQzLXMXRfaq9FVcPJc"
 
 FEE_AMOUNT = "1000000000000" -- 1 $0RBT
-BASE_URL = "https://dummyjson.com/products"
+BASE_URL = "https://api.diadata.org/v1/assetQuotation/Arweave/0x0000000000000000000000000000000000000000"
```

Comment:
any reason for not using coingecko api?

Reply:
yes, it had some issues with the free api and hence used this instead.
https://docs.coingecko.com/reference/simple-price